h/static/scripts/annotator/annotator.coffee

```diff
@@ -329,6 +297,7 @@ class Annotator extends Delegator
   # Returns annotation Object.
   updateAnnotation: (annotation) ->
     this.publish('beforeAnnotationUpdated', [annotation])
+    $(annotation.highlights).attr('data-annotation-id', annotation.id)
```

Comment:
`annotation.highlights` is not a thing on our branch.
Instead, we have `annotation.anchors`, which is an array, and then each element has a `highlight` field, which is a map, for pageNumber -> highlight node.